src/application/CMakeLists.txt

```diff
@@ -23,15 +24,19 @@ endif()
 # Define f3d target
 add_executable(f3d ${F3D_SOURCE_FILES})
 target_link_libraries(f3d PUBLIC libf3d)
+
+if (UNIX AND NOT APPLE)
```

Comment:
I think clang does not need it. Maybe you should test if the compiler is GCC instead

Reply:
it is needed:

```
 [100%] Linking CXX executable ../../bin/f3d
CMakeFiles/f3d.dir/F3DOptionsParser.cxx.o: In function `std::filesystem::__cxx11::path::path<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::filesystem::__cxx11::path>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::filesystem::__cxx11::path::format)':
/usr/bin/../lib/gcc/x86_64-linux-gnu/8/../../../../include/c++/8/bits/fs_path.h:185: undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
CMakeFiles/f3d.dir/F3DOptionsParser.cxx.o: In function `ConfigurationOptions::InitializeDictionaryFromConfigFile(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)':
```